lib/applicationinsights.json

```diff
@@ -1,4 +1,5 @@
 {
+  "connectionString": "${file:/mnt/secrets/draft-store/app-insights-connection-string}",
   "role": {
     "name": "rpe-demo"
```

Comment:
This role name needs to be updated to something DARTS related I think

Reply:
I think I'll need to add the required Azure resources before attempting to connect or pass anything to them. That's why I left the application part for another commit. Thanks